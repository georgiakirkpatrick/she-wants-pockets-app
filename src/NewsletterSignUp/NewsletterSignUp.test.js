import React from 'react';
import ReactDOM from 'react-dom';
import NewsletterSignup from './NewsletterSignup';
import renderer from 'react-test-renderer';

describe ('NewsletterSignup', () => {
    it ('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<NewsletterSignup />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it('renders the UI as expected', () => {
        const tree = renderer
            .create(<NewsletterSignup />)
            .toJSON()
        expect(tree).toMatchSnapshot()    
    })
})