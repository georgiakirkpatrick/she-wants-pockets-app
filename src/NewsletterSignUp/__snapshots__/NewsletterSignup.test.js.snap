// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsletterSignup renders the UI as expected 1`] = `
<form
  className="NewsletterSignUp"
>
  <label
    htmlFor="newsletter"
  >
    Want updates?
  </label>
  <input
    id="newsletter"
    name="newsletter"
    type="text"
  />
  <button
    type="button"
  >
    Submit
  </button>
</form>
`;
